test/test_io_raster.py

```diff
@@ -1069,6 +1066,22 @@ def test_read_raster(path):
     assert not rr.data.mask.all()
 
 
+@pytest.mark.aws_s3
```

Comment:
`aws_s3` is only for tests which access `s3://mapchete-test` testdata. These fixtures only use data from our docker based integration setup, so marking them with `integration` is sufficient.

Reply:
there is one `raster_4bands_s3` so this somewhat applies only for 1 param there, and would probably not split this even further